src/components/StackNameGenerator.jsx

```diff
@@ -30,13 +29,12 @@ class StackNameGenerator extends React.Component {
     });
   }
 
-  // maybe we'll memoize this in the future
-  getData = memoize(this.makeRequest);
-
   async generateStack(stackName) {
     let listOfThings = [];
     for (var i = 0; i < stackName.length; i++) {
-      if (/^[a-zA-Z]+$/.test(stackName.charAt(i))) {
+      if(this.state.generatedList.length > i && this.state.generatedList[i].charAt(0).toLowerCase() === stackName.charAt(i).toLowerCase()) {
```

Comment:
use generatedlist to determine if a value at a specific index was generated during the last iteration